plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/submitit_launcher.py

```diff
@@ -100,37 +103,36 @@ def launch(
 
         num_jobs = len(job_overrides)
         assert num_jobs > 0
-        exec_name = self.params.executor.value
         params = self.params
 
         # build executor
-        init_renamer = dict(executor="cluster", submitit_folder="folder")
+        init_renamer = dict(submitit_folder="folder")
         init_params = {name: params[k] for k, name in init_renamer.items()}
         init_params = {
             k: v.value if isinstance(v, Enum) else v for k, v in init_params.items()
         }
         specific_init_keys = {"max_num_timeout"}
         init_params.update(
             **{
-                f"{exec_name}_{x}": y
+                f"{self._EXECUTOR}_{x}": y
                 for x, y in params.items()
                 if x in specific_init_keys
             }
         )
         init_keys = specific_init_keys | set(init_renamer)  # used config keys
-        executor = submitit.AutoExecutor(**init_params)
+        executor = submitit.AutoExecutor(cluster=self._EXECUTOR, **init_params)
 
         # specify resources/parameters
         baseparams = set(dataclasses.asdict(BaseParams()).keys())
         params = {
-            x if x in baseparams else f"{exec_name}_{x}": y
+            x if x in baseparams else f"{self._EXECUTOR}_{x}": y
             for x, y in params.items()
             if x not in init_keys
         }
         executor.update_parameters(**params)
 
         log.info(
-            f"Submitit '{exec_name}' sweep output dir : "
+            f"Submitit '{self._EXECUTOR}' sweep output dir : "
             f"{self.config.hydra.sweep.dir}"
         )
         sweep_dir = Path(str(self.config.hydra.sweep.dir))
```

Comment:
I hope this part can be simplified once you make some changes to submitit.

Reply:
It would get only marginally simpler: getting rid of the need for baseparams.
Then again it's not that complicated, and that should be robust.

Reply:
ok.
We can potentially simplify by moving the @dataclass into submitit later. definitely good enough for now.